File: raindrops.py
def convert(number):
    sound = ""
    has_factor = False
    if number % 3 == 0:
        sound += "Pling"
        has_factor = True
    if number % 5 == 0:
        sound += "Plang"
        has_factor = True
    if number % 7 == 0:
        sound += "Plong"
        has_factor = True
    if not has_factor:
        sound = str(number)
    return sound

File: kotlin/Raindrops.kt
object Raindrops {

    fun convert(n: Int): String {
        var soundString = ""
        if (n % 3 == 0){
            soundString += "Pling"
        }
        if (n % 5 == 0){
            soundString += "Plang"
        }
        if (n % 7 == 0){
            soundString += "Plong"
        }
        return if (soundString == ""){
            n.toString()
        }else{
            soundString
        }
    }
}
